packages/web/src/components/nav/desktop/AccountDetails.tsx

```diff
@@ -1,66 +1,130 @@
 
   return (
-    <div className={styles.userHeader}>
-      <div className={styles.accountWrapper}>
-        <AvatarLegacy userId={account?.user_id} />
-        <div className={styles.userInfo}>
-          {account ? (
-            <>
-              <UserLink
-                textVariant='title'
-                size='s'
-                strength='weak'
-                userId={account.user_id}
-                badgeSize='xs'
-              />
-              <TextLink
-                size='xs'
-                to={profileLink}
-              >{`@${account.handle}`}</TextLink>
-            </>
-          ) : (
-            <>
-              <Text variant='body' size='s' strength='strong'>
-                {messages.haveAccount}
-              </Text>
-              <TextLink
-                to={SIGN_IN_PAGE}
-                variant='visible'
-                size='xs'
-                strength='weak'
-              >
-                {messages.signIn}
-              </TextLink>
-            </>
-          )}
-        </div>
-        {isManagerModeEnabled && account ? <AccountSwitcher /> : null}
-      </div>
-    </div>
+    <Flex direction='column' pb='unit5' w='100%'>
+      {isManagedAccount ? (
+        <Box pv='xs' ph='m' backgroundColor='accent'>
+          <Text variant='label' size='xs' color='staticWhite'>
+            {messages.managedAccount}
+          </Text>
+        </Box>
+      ) : null}
+      <Flex
+        pt='l'
+        pr='s'
+        pb='s'
+        pl='m'
+        css={{
+          ...(isManagedAccount
+            ? {
+                '&:before': {
+                  content: '""',
+                  position: 'absolute',
+                  top: 0,
+                  left: 0,
+                  height: '100%',
+                  width: '100%',
+                  backgroundColor: color.background.accent,
+                  opacity: 0.03
+                },
+                borderBottom: `1px solid ${color.border.strong}`
+              }
+            : {})
+        }}
+      >
+        <Flex alignItems='center' w='100%' justifyContent='flex-start' gap='l'>
+          <AvatarLegacy userId={account?.user_id} />
+          <Flex
+            direction='column'
+            gap='xs'
+            flex={1}
+            css={{
+              textAlign: 'left',
+              whiteSpace: 'nowrap',
+              wordBreak: 'keep-all',
+              overflow: 'hidden'
+            }}
+          >
+            {account ? (
+              <>
+                <Flex
+                  alignItems='center'
+                  justifyContent='space-between'
+                  gap='l'
+                >
+                  <UserLink
+                    textVariant='title'
+                    size='s'
+                    userId={account.user_id}
+                    badgeSize='xs'
+                    css={{
+                      flex: 1,
+                      ...(isManagedAccount && {
```

Comment:
note to self to add this as a variant to user-link

Reply:
I thought about it, but Julian seemed to indicate this was a one-off